unicorn.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   rules: {
-    "unicorn/number-literal-case": "off"
+    "unicorn/number-literal-case": "off",
```

Comment:
The rules in these files are sort alphabetically :)

```suggestion
    "unicorn/no-nested-ternary": "off",
```